src/site/includes/header.html

```diff
@@ -60,7 +60,7 @@
 <![endif]-->
 
   {% if botframework_cdn %}
-    <script src="{{ botframework_cdn }}"></script>
+    <script nonce="**CSP_NONCE**" src="{{ botframework_cdn }}"></script>
```

Comment:
I don't think a nonce is necessary on scripts containing a `src`, but doesn't hurt

Reply:
`strict-dynamic` causes all domain whitelisting `self` and `unsafe-inline` to be ignored

Reply:
Yeah - I'm experimenting with this an another script as well.